_docs/12-component-controllers.md

```diff
@@ -33,4 +33,48 @@ But sometimes, you do need an object with a longer lifecycle. *Component control
 
 There is a only a one-way communication channel between the component and its component controller - you can only pass data off of a component to a component controller. A component has no reference its corresponding component controller. This is by design. 
```

Comment:
> A component has no reference its corresponding component controller.

Actually not true. component.controller is the component's associated controller. We *discourage* accessing it, and *forbid* access to it inside the component's initializer via an assertion, and return nil if access is attempted.